### PR TITLE
Detects correct muc service based on prefix "conference".

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: src/main/java/org/jitsi/jicofo/FocusManager.java [hunk 1]
```diff
@@ -289,7 +300,9 @@ public void start()
             = new JitsiMeetServices(
                     protocolProviderHandler,
                     jvbProtocolProvider,
-                    focusUserDomain);
+                    focusUserDomain,
+                    JidCreate.domainBareFrom(
+                        conferenceMucPrefix + "." + xmppDomainConfig));
```
Comment (bgrozev) [inline src/main/java/org/jitsi/jicofo/FocusManager.java:305]: If we're matching the complete domain, why not get rid of the discovery logic? Just pass the configuration to `Health`.

Comment (paweldomas) [reply src/main/java/org/jitsi/jicofo/FocusManager.java:305]: +1

Comment (damencho) [reply src/main/java/org/jitsi/jicofo/FocusManager.java:305]: Ok, make sense :)

Comment (damencho) [reply src/main/java/org/jitsi/jicofo/FocusManager.java]: How about now? 
Now this PR removes more than it adds :)

**Pull request merged**